### PR TITLE
LJ-24  setting up the canada.ca footer

### PR DESCRIPTION
**Pull request by MarcoGoC**
# Description
  This is just the footer from the canada.ca theme that I did for another project a while back 
  it has more than actually needed css, but not even as close as the 600Kb css files of the canada.ca website    

Footer, landmark, woodmark, deleted obsolete file and updated the global.css 

## Test Instructions

1. Pull the branch and run 
2. Footer should work correctly in full medium and small screens resolutions
3. there are no tests 

## Meets Definition of Done

- [ ] TODO: Add Criteria

### CODE REVIEW

File: components/atoms/Woodmark.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import Link from "next/link";
+
```
Comment (pcraig3) [inline components/atoms/Woodmark.js:2]: I think this is called a `Wordmark`. :)

File: pages/joinroom.js [hunk 2]
```diff
@@ -1,38 +0,0 @@
-import Image from "next/image";
-import Link from "next/link";
```
Comment (pcraig3) [inline pages/joinroom.js:2]: 👍  yeah, cool, let's get rid of this.

**Review by pcraig3 — COMMENTED** [review]
Is the CSS you've added going to apply to the header and layout as well? Or is it just for the footer?

**Review by pcraig3 — COMMENTED** [review]
Just built this locally and it looks super good.

File: locales/en.js [hunk 3]
```diff
@@ -66,6 +66,9 @@ export default {
   privacyLink: "https://www.canada.ca/en/transparency/privacy.html",
   privacyText: "Privacy",
```
Comment (krischarbonneau) [inline locales/en.js:68]: The link for prime minister currently redirects to http://localhost:3000/pm.gc.ca/en. Works as intended if you change the link to `http://pm.gc.ca/en`

Same thing with the french link

**Review by krischarbonneau — APPROVED** [review]
Looks fantastic. Great job Marco

**Review by pcraig3 — APPROVED** [review]
I think this looks great! Let's get it in.

File: components/atoms/Woodmark.js (hunk 1 above)
Comment (MarcoGoC) [reply components/atoms/Woodmark.js:2]: that makes more sense

**Pull request merged**